src/contract-details/ContractDetailsCard.js

```diff
@@ -18,54 +19,28 @@ export default class ContractDetailsCard extends Component {
 		const workAroundForBackendBugIsDigitTrade = contract.shortcode && contract.shortcode.includes('DIGIT');
 		const validToSell = contract.is_valid_to_sell === 1 && !contract.is_expired;
 		const validationError = contract.validation_error;
-		const potentialProfit = contract.payout - contract.buy_price;
-		const profit = sold && (contract.sell_price - contract.buy_price);
+		const potentialProfit = (contract.payout - contract.buy_price).toFixed(2);
```

Comment:
Haha :)
It is more to differentiate much more easily the usage of say 'you have $20.00' and 'this stock is $33.0023'.

Actually, this is probably incorrect assumption on your side. Japanese Yen maybe does not have significant numbers after the decimal point. Also, bitcoins are used with more symbols.